pkg/account/info.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package account
+
+type AccountInfo struct {
+	Name        string
+	AccountUUID string
+}
+
+func (a AccountInfo) String() string {
+	if len(a.Name) > 0 {
```

Comment:
nit: `a.Name == ""` feels like the more idiomatic way to check if a string is empty. 

also: should we maybe log it as "name (uuid)" if both are set, rather than just the name?